src/tactic/ui/app/notify_wdg.py

```diff
@@ -25,57 +25,81 @@ class NotifyWdg(BaseRefreshWdg):
     def get_display(my):
 
         top = my.top
-
         top.set_id("spt_notify_top")
 
-        top.add_style("position: fixed")
-        top.add_style("top: 60px")
-        top.add_style("z-index: 10000")
-        top.add_style("width: 200px")
-        top.add_style("text-align: center")
-        top.add_style("margin-top: -100px")
+        inner = DivWdg()
+        top.add(inner)
+
+        inner.set_class("spt_notify_el")
+
+        inner.add_style("position: fixed")
+        inner.add_style("top: 60px")
+        inner.add_style("z-index: 10000")
+        inner.add_style("width: auto")
+        inner.add_style("text-align: center")
+        inner.add_style("margin-top: -100px")
 
-        top.add_color("background", "background", -3)
-        top.add_style("height: 20px")
-        top.add_style("padding: 5px")
-        top.add_border()
+        inner.add_color("background", "background", -3)
+        inner.add_style("height: auto")
+        inner.add_style("padding: 10px 20px")
+        inner.add_border()
 
 
         msg_div = DivWdg()
-        top.add(msg_div)
+        inner.add(msg_div)
         msg_div.add_class("spt_notify_message")
 
 
 
 
         top.add_behavior( {
             'type': 'load',
-            'cbjs_action': '''
+            'cbjs_action': r'''
 spt.notify = {};
 
 spt.notify.last_settings = {};
 
-spt.notify.show = function() {
-    new Fx.Tween('spt_notify_top').start('opacity', 1);
-    new Fx.Tween('spt_notify_top').start('marginTop', 0);
+spt.notify.top = bvr.src_el;
+
+spt.notify.clone_el = null;
+
+spt.notify.clone = function() {
+    var template = spt.notify.top.getElement(".spt_notify_el");
+    var clone = spt.behavior.clone( template );
```

Comment:
If you did set the display to none like my suggestion, you may need to run
spt.show(clone) here